contrib/swww/template.py

```diff
@@ -0,0 +1,12 @@
+pkgname = "swww"
+pkgver = "0.8.1"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = [ "cargo" ]
+makedepends = [ "rust" ]
```

Comment:
CI is waiting on approval but the [`black`](https://pkgs.chimera-linux.org/package/current/main/x86_64/python-black) job will fail these lines. You'll need to format the template with it.